README.md

```diff
@@ -114,10 +119,16 @@ For your convenience, Squiss provides direct access to the AWS SDK's SQS object,
 
 ## Events
 
+### deleted {Message}
+Emitted when a message is confirmed as being successfully deleted from the queue. The `handled` and `delQueued` events will also be fired for deleted messages, but that will come earlier, when the delete function is initially called.
```

Comment:
Excellent question! When `handled` is emitted, Squiss is indicating that it's removed the message from its own in-flight status -- which, if a delete or release ultimately fails, is _still accurate_. When one of those things fails, an `error` event is emitted which is literally impossible to ignore. Either you have a listener for it, or Node crashes your app.

I could see an argument for a `preRelease` versus `postRelease` event so you don't have to guess how a message was `handled`, but at this point no use cases have turned up for that level of verbosity.